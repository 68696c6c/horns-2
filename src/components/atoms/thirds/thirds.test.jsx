/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import Thirds from '.'

describe('Thirds', () => {
  it('should render as default', () => {
    const { container } = render(
      <Thirds>
        <div>a</div>
        <div>b</div>
      </Thirds>
    )
    expect(container).toMatchSnapshot()
  })
})
