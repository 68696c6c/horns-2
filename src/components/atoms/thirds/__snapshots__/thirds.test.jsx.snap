// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Thirds should render as default 1`] = `
.emotion-0 {
  display: block;
  grid-template-columns: 1fr 2fr;
  grid-template-areas: 'left right';
}

@media (min-width:480px) {
  .emotion-0 {
    display: grid;
  }
}

@media (min-width:1200px) {
  .emotion-0 > :nth-child(1) {
    padding-left: calc(((100vw - 1200px) / 2));
  }

  .emotion-0 > :nth-last-child(1) {
    padding-right: calc(((100vw - 1200px) / 2));
  }
}

.emotion-0 > :nth-child(odd) {
  grid-area: left;
}

.emotion-0 > :nth-child(even) {
  grid-area: right;
}

<div>
  <div
    class="emotion-0"
  >
    <div>
      a
    </div>
    <div>
      b
    </div>
  </div>
</div>
`;
